guide/payments/request.md

```diff
@@ -80,33 +68,17 @@ Below is an overview of these various entry points:
 
 ## Single-use payment requests
 
-The most common way bitcoin is requested today is in the form of single-use payment requests. For Lightning, a standard [BOLT11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md) invoice is used. For on-chain, [addresses of various types]({{ "/guide/glossary/address/" | relative_url }}) are used. Addresses can be reused though this is not advised due to [privacy concerns]({{ "/guide/payments/privacy/#generate-a-new-address-for-each-on-chain-payment" | relative_url }}). Single-use invoices and addresses need to be easily and conveniently generated within a wallet application.
-
-### Amount
-
-When generating a single-use payment request, the receiver may want to add an amount to be paid. Entering an amount from the home screen can speed up the requesting flow. Alternatively, users can add an amount to an already generated payment request. Entering an amount needs to be clear to prevent users entering incorrect amounts.
-
-Amount units need to be toggleable between bitcoin, satoshi and local currency values as senders may request in a set unit. Ensure the unit being entered is prominent to prevent entering of the wrong unit amount. More on the [Units & Symbols page]({{ "/guide/payments/units-and-symbols/" | relative_url }}).
-
-### Address types
-
-There are several address types that can be used for on-chain payment requests. Each address type has unique benefits, such as SegWit saving on network fees. It's recommended to encourage users to use the latest - more feature rich address types. However, older address types should be supported for seamless backwards compatibility. More on the [address types]({{ "/guide/glossary/address/" | relative_url }}) page.
+The most common way bitcoin is requested is in the form of single-use payment requests. Lightning payments use [BOLT11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md) invoices. On-chain uses [addresses of various types]({{ "/guide/glossary/address/" | relative_url }}). These two formats can be combined into a single payment request so senders can use what is most convenient for them.
```

Comment:
Should we provide detail that this is a BIP21 payment link with added BOLT 11 (and in future BOLT 12) invoice?

Reply:
There are two ways to combine thiese:
- A BIP21 URI with `&lightning=` parameter
- A BOLT11 invoice with an on-chain address fallback

Can a BOLT11 invoice also turned into a clickable link or button that browsers and apps pick up? If not, then we have two slightly different use cases.

Reply:
the `lightning:` URI is recognized by some Lightning apps and is used for BOLT11 invoices so yes.

Reply:
@GBKS the advantage that the BIP21 QR has is that it's automatically backwards compatible with any on-chain wallet. BIP21 has been around a while, so likely most on-chain wallets support this format. Sure, it looks like a URI for a clickable link, but it doesn't have to be used that way.